modules/etcd-certificates.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/certificate-reference.adoc
+
+[id="etcd-certificates_{context}"]
+= etcd certificates
+
+[discrete]
+== Purpose
+
+etcd certificates are signed by the etcd-signer; they come from a certificate
+authority (CA) that is generated by the bootstrap process.
+
+etcd certificates are used for encrypted communication between etcd member peers,
+as well as encrypted client traffic. The following list of certificates are
+generated and used by etcd and other processes communicating with etcd:
+
+* Peer certificates: Used for communication of etcd members.
+* Client certificates: Used for encrypted server-client communication. Client
+certficates are currently used by the API server only, and no other service
+should connect to etcd directly except for the proxy.
+* Server certificates: Used by etcd server for authenticating client requests.
+* Metric certificates: All metric consumers connect to proxy with metric-client
+certificates.
+
+[discrete]
+== File path
+
+* CA certificates:
+** etcd CA certificate: `/etc/ssl/etcd/ca.crt`
+** etcd metric CA certificate: `/etc/ssl/etcd/metric-ca.crt`
+* Server certificates: `/etc/ssl/etcd/system:etcd-server`
+* Client certificates: `APISERVER_POD_DIR/secrets/etcd-client/`
+* Peer certificates: `/etc/ssl/etcd/system:etcd-peer`
+* Metric certificates: `/etc/ssl/etcd/metric-signer`
+
+[discrete]
+== Expiration
+
+The CA certificates are valid for 10 years. The peer, client, and server
+certificates are valid for 3 years.
+
+[discrete]
+== Services
+
+* Peer certificates are used by etcd members for peer communication among themselves.
```

Comment:
This list seems like it repeats what was already said above in the purpose. Could it be here and not up there?

Reply:
Good catch. I meant to move that list.